app/models/concerns/pageflow/reusable_file.rb

```diff
@@ -56,14 +56,66 @@ def cache_key
       "#{super}-#{state}"
     end
 
+
+    # The following are method defaults for file types that do not require processing/encoding.
+    # They are overwritten with default values in UploadableFile for files that are uploaded
+    # through the editor.
+
+    # Overwritten if a file type provides a default url for its file.
+    def url
+      ''
+    end
+
+    # Overwritten in case of a file type providing its original (unprocessed) file
+    # for download in the editor.
+    # Defaults to the default url of the file (see above)
+    def original_url
+      url
+    end
+
+    # Overwritten with the basename of the file.
+    def basename
+      'unused'
+    end
+
+    # Overwritten with the actual attachment of the file type.
+    # Unifies the handling of attachments throughout the codebase by relying on
+    # `attachment` always being present regardless of its actual name.
+    def attachment
+      require 'ostruct'
+      OpenStruct.new(original_file: 'unused')
+    end
```

Comment:
Where is this use? File types like `Chart::ScrapredSite` do not have a single attachment.